terraform/install.sh

```diff
@@ -253,6 +253,13 @@ getExternalIp() {
 # Install Load Generator service and start generating synthetic traffic to Sandbox
 loadGen() {
   log "Running load generator"
+  # remove existing load generator
+  old_loadgen=$(gcloud compute instances list --project $project_id --filter="name:loadgenerator*" --format="value(name)")
+  if [[ -n "$old_loadgen" ]]; then
+    loadgen_zone=$(gcloud compute instances list --project $project_id --filter="name:loadgenerator*" --format="value(zone)")
+    $(gcloud compute instances delete $old_loadgen --zone $loadgen_zone --quiet)
```

Comment:
This seems like a bit of a hack. Instead of deleting the old one, can't we just only create a new one if `old_loadgen` doesn't exist?

```
loadgen_ip=$(gcloud compute instances list --project "$project_id" \
                                           --filter="name:loadgenerator*" \
                                           --format="value(networkInterfaces[0].accessConfigs.natIP)")
if [[ -z "$loadgen_ip" ]]; then
    # load generator not found. Launch a new one
  ../loadgenerator/loadgen autostart $external_ip
fi

while [[ $(curl -sL -w "%{http_code}"  "http://$loadgen_ip:8080" -o /dev/null --max-time 1) -ne 200  && \
...
```

Reply:
Or is the issue that it might be pointing to the wrong external IP?

Reply:
@simonz130 This was pointed out by you in the issue. I'm also a bit confused. Can you explain why? Thanks!

Reply:
When reusing the Sandbox which might already have the loadgen provisioned, terraform will re-provision loadgen so we'll end up having two and both IPs will be reported upon completion of provisioning script.
What we need is a single loadgen. 

So two options:

1) Remove the "old" loadgen and provision a new one
2) Keep the "old" one and don't provision the new one

If we go with (2), it might be running against demo cluster or it might be that demo cluster now has a new IP so "old" loadgen needs to be restarted against the new IP of the FE. As long as we ensure that the end result is that loadgen is running against the correct IP of the FE - I'm fine.

If we go with (1), it's simpler - we remove the loadgen - no matter what it does currently and start a new one against the right IP.

For simplicity I prefer (1) but I'm also OK if we do (2). We are planning to replace current loadgen with Cloud Run, so we'll need to revisit this logic anyway.